CHANGELOG.md

```diff
@@ -3,7 +3,7 @@
 ## VNext
 
 - [Fix PropertyFetcher error when used with multiple types](https://github.com/microsoft/ApplicationInsights-dotnet/issues/2194)
-- [New Task Based Flush API - FlushAsync] (https://github.com/microsoft/ApplicationInsights-dotnet/issues/1743)
+- [New Task Based Flush API - FlushAsync](https://github.com/microsoft/ApplicationInsights-dotnet/issues/1743)
```

Comment:
Are you planning to create new PR for change log update. Currently FlushAsync is in vnext and not in2.18.0-beta1

Reply:
yes, i've got one more change coming.